tests/helics/system_tests/ErrorTests.cpp

```diff
@@ -584,8 +584,8 @@ TEST_F(error_tests, missing_required_ept)
     auto fed2 = GetFederateAs<helics::MessageFederate>(1);
 
     fed1->registerGlobalTargetedEndpoint("t1", "");
-    auto& e2 = fed2->registerGlobalTargetedEndpoint("abcd", "");
-    e2.setOption(helics::defs::Options::CONNECTION_REQUIRED);
+    auto& ept2 = fed2->registerGlobalTargetedEndpoint("abcd", "");
+    ept2.setOption(helics::defs::Options::CONNECTION_REQUIRED);
```

Comment:
```suggestion
    auto& ept1 = fed2->registerGlobalTargetedEndpoint("abcd", "");
    ept1.setOption(helics::defs::Options::CONNECTION_REQUIRED);
```

Since it's the only ept in this function, to match the previous test case where i2 was renamed sub1?